system/mp/mpcomm.cpp

```diff
@@ -824,7 +825,7 @@ protected: friend class CMPPacketReader;
             {
                 StringBuffer str;
 #ifdef _TRACE
-                LOG(MCdebugInfo(100), unknownJob, "MP: connecting to %s",remoteep.getUrlStr(str).str());
+                LOG(MCdebugInfo(100), unknownJob, "MP: connecting to %s role: %llu", remoteep.getUrlStr(str).str(), parent->getRole());
```

Comment:
should use %" I64F "u

Reply:
Yes, of course, since it was in a debug _TRACE I didn't bother.  I will fix.